src/getOperands.js

```diff
@@ -26,7 +26,9 @@
  * @return {string} - operands
  */
 function getOperands(str) {
-  // write code here
+  const result = str.match(/[+/*-]*\d+\.*\d*/g);
```

Comment:
match возвращает совпадения (matches), паттерн в переменную
```suggestion
  const pattern = /[+/*-]*\d+\.*\d*/g;
  const matches = str.match(pattern);
```